intranet/apps/dataimport/management/commands/import_eighth.py

```diff
@@ -0,0 +1,66 @@
+import json
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand, CommandError
+from django.db.models import Q
+
+from intranet.apps.eighth.models import EighthActivity, EighthRoom, EighthSponsor
+
+
+class Command(BaseCommand):
+    help = "Import Fake Activities"
+
+    def add_arguments(self, parser):
+        # Positional arguments
+        parser.add_argument("data_fname")
+
+    def handle(self, *args, **kwargs):
+        try:
+            with open(kwargs["data_fname"]) as f_obj:
+                data = json.load(f_obj)
+        except OSError as ex:
+            raise CommandError(str(ex)) from ex
+
+        for activity in data["activities"]:
+            name = activity["Name"].strip()
+            description = activity["Description"].strip()
+            sponsor_first_name, sponsor_last_name = activity["Sponsor"].strip().split()
+            room_num = activity["Room Number"]
+            capacity = activity["Capacity"]
+            date = activity["Date"].strip()
+            classification = activity["Classification"]
+
+            if len(sponsor_last_name) > 7:
+                sponsor_username = sponsor_first_name[:1] + sponsor_last_name[:7]
+            else:
+                sponsor_username = sponsor_first_name[:1] + sponsor_last_name[:7]
+
+            if not get_user_model().objects.filter(Q(username=sponsor_username)).exists():
```

Comment:
In production, many activities support multiple sponsors. Is that feasible to support in this fake data generation script?